.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+🎉 Thank you for submitting a PR! Before submitting, please review the following checklist:
+
+- [ ] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](/docs/CONTRIBUTING.md)
+- [ ] ✅ I've added tests where applicable to test my new functionality.
+- [ ] 📝 I've followed the Solidity and JavaScript style guides.
+- [ ] 📖 I've made sure that my contracts are well-documented.
+- [ ] 🎨 I've run the JavaScript linter (`npm run lint`) and fixed all issues.
+
+**Does this close any open issues?** If so, list them here.
```

Comment:
The problem with this is that the contributor will not write a phrase like "fixes #PR" which is [what GitHub uses](https://help.github.com/articles/closing-issues-using-keywords/) to close said issue automatically when the PR is merged. How can we nudge the contributor towards that?

> **Does this close any open issues?** I so, list them here.
> Fixes #...